nilearn/plotting/displays.py

```diff
@@ -1216,6 +1221,9 @@ def add_graph(self, adjacency_matrix, node_coords,
                               vmin=edge_vmin, vmax=edge_vmax,
                               **edge_kwargs)
 
+        if colorbar:
+            self._colorbar_show(ax, threshold=edge_threshold)
```

Comment:
>Basically you noticed that the im argument of _colorbar_show only needed a .cmap and a >.norm and you defined it appropriately on the GlassBrainAxes instance so that everything just > works when you pass a GlassBrainAxes instance to _colorbar_show.

Yes